pkg/ssh/ssh_test.go

```diff
@@ -235,11 +235,11 @@ func hasCorrectTunnels(tunnelList *SSHTunnelList, addresses []string) bool {
 		wantMap[addr] = true
 	}
 	haveMap := make(map[string]bool)
```

Comment:
```
wantMap := sets.NewString(adresses...)
haveMap := sets.StringKeySet(tunnelList.tunnels)
return wantMap.Eq(haveMap)
```
https://github.com/kubernetes/kubernetes/blob/43ce2f17cf81907f1142e0e41f76c53f09f67942/staging/src/k8s.io/apimachinery/pkg/util/sets/string.go#L29-L46

Reply:
cool, it's much simpler. Thanks.